frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/PipelineForm.tsx

```diff
@@ -58,8 +58,9 @@ const PipelineForm: React.FC<PipelineFormProps> = ({
         onSubmit={handleSubmit}
         onReset={handleReset}
         validationSchema={validationSchema}
-        render={(props) => <PipelineFormComponent namespace={obj.metadata.namespace} {...props} />}
-      />
+      >
+        {(props) => <PipelineFormComponent namespace={obj.metadata.namespace} {...props} />}
```

Comment:
Can we make it consistent across all Formik forms? Right now it says `props`, `formProps` and `formikProps` in various files.
```suggestion
        {(formikProps) => <PipelineFormComponent namespace={obj.metadata.namespace} {...formikProps} />}
```

Reply:
I like that. Will change it to `formikProps` okay? But only code which are effected by this commits.

Reply:
@rohitkrai03 I have done this. I'm unsure if this makes the change "too big", but all <Formik children-render functions uses now `formikProps`... What do you think?